lib/common/strings.c

```diff
@@ -17,7 +17,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include <ctype.h>
+#include <float.h>  // DBL_MIN
 #include <limits.h>
+#include <math.h>   // fabs()
```

Comment:
Thanks for the heads-up there. I'll try to keep that in mind should it become relevant :)

> the new name is CRM_SCORE_INFINITY

Surprised we're not using a PCMK prefix for that too.

Reply:
The approach has evolved over the years. CRM_SCORE_INFINITY was part of an early attempt to use existing prefixes more consistently. Since then we've decided on PCMK (eventually for everything, but starting with new or internal stuff in libcrmcommon and libpacemaker).

My goal is to rename all internal stuff first, then eventually rename public API with the legacy names as compatibility wrappers. Coccinelle should make that easier. For example this case would become

    #define PCMK_SCORE_INFINITY 1000000
    ...
    // Begin compatibility section not accessible by pacemaker code
    ....
    //! \deprecated Use PCMK_SCORE_INFINITY instead
    #define CRM_SCORE_INFINITY PCMK_SCORE_INFINITY